package.json

```diff
@@ -32,12 +32,14 @@
   "license": "MIT",
   "dependencies": {},
   "devDependencies": {
+    "browserify": "^13.1.1",
+    "tsify": "^2.0.7",
     "@types/pixi.js": "^4.3.1",
     "typescript": "^2.1.4"
   },
   "scripts": {
     "demo": "npm run build && open demo/index.html",
-    "build": "tsc",
+    "build": "tsc; browserify pixi-multistyle-text.ts -p tsify -s MultiStyleText -o dist/pixi-multistyle-text.js -d",
```

Comment:
I found a solution.  I don't love it (it's kind of a hack), but it works.  See TypeStrong/tsify#105.